tests/test_utils.py

```diff
@@ -0,0 +1,132 @@
+    """A test case for testing sky position offsets for exposures."""
+
+    def setUp(self):
+        camera = Latiss.getCamera()
+        self.assertTrue(len(camera) == 1)
+        self.detector = camera[0]
+
+        self.viMaker = MakeRawVisitInfoViaObsInfo()
+        self.mi = afwImage.maskedImage.MaskedImageF(0, 0)
+        self.baseHeader = dict(boresight_airmass=1.5,
+                               temperature=15*u.deg_C,
+                               observation_type="science",
+                               exposure_time=5*u.ks,
+                               detector_num=32,
+                               )
+
+    def test_getExpPositionOffset(self):
+        epsilon = 0.0001
+        ra1s = [0, 45, 90]
+        ra2s = copy.copy(ra1s)
+        ra2s.extend([r + epsilon for r in ra1s])
+        ra1s = np.deg2rad(ra1s)
+        ra2s = np.deg2rad(ra2s)
+
+        epsilon = 0.0001
+        dec1s = [0, 45, 90]
+        dec2s = copy.copy(dec1s)
+        dec2s.extend([d + epsilon for d in dec1s[:-1]])  # skip last point as >90 not allowed for dec
+
+        rotAngle1 = geom.Angle(43.2, geom.degrees)  # arbitrary non-zero
+        rotAngle2 = geom.Angle(56.7, geom.degrees)
+
+        t1 = astropy.time.Time("2021-09-15T12:00:00", format="isot", scale="utc")
+        t2 = astropy.time.Time("2021-09-15T12:01:00", format="isot", scale="utc")
+        expTime = astropy.time.TimeDelta(20, format='sec')
+
+        header1 = copy.copy(self.baseHeader)
+        header2 = copy.copy(self.baseHeader)
+        header1['datetime_begin'] = astropy.time.Time(t1, format="isot", scale="utc")
+        header2['datetime_begin'] = astropy.time.Time(t2, format="isot", scale="utc")
+
+        header1['datetime_end'] = astropy.time.Time(t1+expTime, format="isot", scale="utc")
+        header2['datetime_end'] = astropy.time.Time(t2+expTime, format="isot", scale="utc")
+
+        obsInfo1 = makeObservationInfo(**header1)
+        obsInfo2 = makeObservationInfo(**header2)
+
+        vi1 = self.viMaker.observationInfo2visitInfo(obsInfo1)
+        vi2 = self.viMaker.observationInfo2visitInfo(obsInfo2)
+        expInfo1 = afwImage.ExposureInfo()
+        expInfo1.setVisitInfo(vi1)
+        expInfo2 = afwImage.ExposureInfo()
+        expInfo2.setVisitInfo(vi2)
+
+        for ra1, dec1, ra2, dec2 in itertools.product(ra1s, dec1s, ra2s, dec2s):
+            pos1 = geom.SpherePoint(ra1, dec1, geom.degrees)
+            pos2 = geom.SpherePoint(ra2, dec2, geom.degrees)
+
+            wcs1 = createInitialSkyWcsFromBoresight(pos1, rotAngle1, self.detector, flipX=True)
+            wcs2 = createInitialSkyWcsFromBoresight(pos2, rotAngle2, self.detector, flipX=True)
+
+            exp1 = afwImage.ExposureF(self.mi, expInfo1)
+            exp2 = afwImage.ExposureF(self.mi, expInfo2)
+
+            exp1.setWcs(wcs1)
+            exp2.setWcs(wcs2)
+
+            result = getExpPositionOffset(exp1, exp2)
+
+            deltaRa = ra1 - ra2
```

Comment:
Again, deltaRa is a red flag without some form of normalization. If you really want it to report a very large delta when they are very close you need to document this.